domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/validation/eventcasefield/EventCaseFieldCasePaymentHistoryViewerCaseFieldValidator.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.hmcts.ccd.definition.store.domain.validation.eventcasefield;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.ccd.definition.store.domain.validation.ValidationResult;
+import uk.gov.hmcts.ccd.definition.store.repository.DisplayContext;
+import uk.gov.hmcts.ccd.definition.store.repository.entity.EventCaseFieldEntity;
+
+@Component
+public class EventCaseFieldCasePaymentHistoryViewerCaseFieldValidator implements EventCaseFieldEntityValidator {
```

Comment:
can't this validator use the same approach used by EventCaseFieldDisplayContextValidatorImpl of defining the exception as an inner static class? 
This way we don't have to have the exception as a new type